CHANGELOG.md

```diff
@@ -3,7 +3,13 @@
 We use PostGraphile's GitHub releases tab to log our changes in detail, but
 this auto-generated changelog helps us to produce that list, and it may be
 helpful to you also.
-# [](https://github.com/graphile/graphile-engine/compare/v4.12.0-alpha.0...v) (2021-02-15)
+# [4.12.0-alpha.1](https://github.com/graphile/graphile-engine/compare/v4.12.0-alpha.0...v4.12.0-alpha.1) (2021-05-12)
```

Comment:
Please pull all version changes back out of the git history (this should mean CHANGELOG, lerna.json, package.json, etc are all unmodified).

Reply:
Done 😄